test/unit/range/views/view_minimiser_test.cpp

```diff
@@ -46,21 +61,39 @@ struct iterator_fixture<iterator_type> : public ::testing::Test
     seqan3::views::minimiser(vec, 5);
 };
 
-using test_type = ::testing::Types<iterator_type>;
-INSTANTIATE_TYPED_TEST_SUITE_P(iterator_fixture, iterator_fixture, test_type, );
+template <>
+struct iterator_fixture<two_ranges_iterator_type> : public ::testing::Test
+{
+    using iterator_tag = std::forward_iterator_tag;
+    static constexpr bool const_iterable = false;
```

Comment:
why not const iterable?

Reply:
Because it leads to an error message, the first error: `error: no match for call to ‘(const ranges::_cbegin_::fn) (seqan3::detail::minimiser_view<seqan3::detail::kmer_hash_view<ranges::ref_view<std::vector<seqan3::dna4> > >, ranges::reverse_view<seqan3::detail::kmer_hash_view<ranges::reverse_view<ranges::transform_view<ranges::ref_view<std::vector<seqan3::dna4> >, seqan3::views::<lambda(auto:12)> > > > > >&)’
         move_forward_post_test<TypeParam>(std::ranges::cbegin(this->test_range)`

Reply:
But do you have a test that tests const iterability with two ranges? I mean, were you actually iterate over the const range?